addons/mail/models/mail_render_mixin.py

```diff
@@ -219,6 +236,46 @@ def _prepend_preview(self, html, preview):
             """).format(preview_markup)
             return tools.prepend_html_content(html, html_preview)
         return html
+
+    # ------------------------------------------------------------
+    # SECURITY
+    # ------------------------------------------------------------
+
+    def _is_dynamic(self):
```

Comment:
can you also update the setting "Restrict Mail Template" to explain the new change ?

Reply:
For me it would make to check it by default indeed